packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss

```diff
@@ -3,6 +3,7 @@
 
     &__content {
         position: relative;
+        display: flex;
```

Comment:
This fixes misalignment for some currencies:
![image](https://github.com/binary-com/deriv-app/assets/103016120/82e5575a-07ee-4922-ae30-6f2020d1e424)